scout/adapter/mongo/query.py

```diff
@@ -268,7 +268,7 @@ def build_query(self, case_id, query=None, variant_ids=None, category="snv"):
 
         # if chromosome coordinates exist in query, add them as first element of the mongo_query['$and']
         if coordinate_query:
-            if mongo_query.get("query"):
+            if mongo_query.get("query") or mongo_query.get("$and"):
```

Comment:
```suggestion
            if mongo_query.get("$and"):
```
It think we can safely remove the first statement here as it is not used

Reply:
Hi @bjhall , thanks for this. Yes the query building part of the code is kind of a nightmare. Not sure how to improve this part. If there are any suggestions from Lund please add them :)